databuilder/databuilder/extractor/es_base_extractor.py

```diff
@@ -129,6 +148,15 @@ def _correct_sort_order(self) -> bool:
         except Exception:
             return False
 
+    # Default time field is @timestamp to match ECS
+    # See https://www.elastic.co/guide/en/ecs/master/ecs-base.html
+    @property
+    def _time_field(self) -> str:
+        try:
+            return self.conf.get(ElasticsearchBaseExtractor.ELASTICSEARCH_TIME_FIELD)
```

Comment:
why not just make @timestamp a default for this config key?

Reply:
Yes, that makes sense to me. Does it make sense to remove the try/catch and just have `@timestamp` as the second parameter?